woodwork/tests/accessor/test_statistics.py

```diff
@@ -1978,6 +1978,15 @@ def test_medcouple_outliers(skewed_outliers_df):
     assert right_skewed_dict == expected_right_skewed_dict
     assert left_skewed_dict == expected_left_skewed_dict
 
+    outliers_series_skewed_right = skewed_outliers_df[
+        "right_skewed_outliers_nullable_int"
+    ]
+    outliers_series_skewed_right.ww.init(logical_type="IntegerNullable")
+
+    right_skewed_dict = outliers_series_skewed_right.ww.medcouple_dict()
+
+    assert right_skewed_dict == expected_right_skewed_dict
```

Comment:
For my understanding, why does this change need to be made?

Reply:
Due to the change made in `_get_box_plot_info_for_column.py`, I'm testing to make sure that an `IntegerNullable` column is appropriately handled.
```
try:
    coeff = np.abs(skew(series))  # This was raising a ValueError for IntegerNullable columns
except ValueError:
    # skew can't handle Int64 dtype
    coeff = np.abs(skew(series.astype("float64")))
```